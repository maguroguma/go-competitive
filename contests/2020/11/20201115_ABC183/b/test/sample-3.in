-9 99 -999 9999
