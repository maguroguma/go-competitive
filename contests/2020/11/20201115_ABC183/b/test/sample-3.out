-18.7058823529
